### PR TITLE
Merge dev to master

### PR DESCRIPTION
**Pull request by jlaneve**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -18,12 +18,12 @@ export default class ImportioSource extends Activity {
 		return getData;
 	}
 
-	async getRawData({ baseURI }) {
+	async getRawData({ import_link }) {
 		const options = {
-			uri: `${baseURI}`
+			uri: `${import_link}`
```
Comment (trong-le) [inline lib/index.js]: @jlaneve Can you fix the indentations? we do 4 spaces as a tab for spacing and indentation along with inside of the readme

**Comment by benjamingregory** [comment]
@trong-le Looks good - can we merge in?

**Comment by trong-le** [comment]
@jlaneve indentation in index.js still needs fixed

**Comment by jlaneve** [comment]
@trong-le my bad, the indentation in my editor was showing up normally. It should be fixed now.

**Pull request merged**